scripts/process-reboot-cause

```diff
@@ -69,6 +71,84 @@ def read_reboot_cause_files_and_save_state_db():
                 x = TIME_SORTED_FULL_REBOOT_FILE_LIST[i]
                 os.remove(x)
 
+def get_dpus():
```

Comment:
@rameshraghupathy can we make use of existing APIs? @vvolam ?

Reply:
@prgeor sure, will do

Reply:
@prgeor done